src/main/java/teamrtg/rtg/core/event/EventManagerRTG.java

```diff
@@ -33,8 +32,15 @@
 import teamrtg.rtg.core.world.gen.structure.MapGenVillageRTG;
 import teamrtg.rtg.core.world.gen.structure.StructureOceanMonumentRTG;
 
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.WeakHashMap;
```

Comment:
@Team-RTG/rtg-writers
We all need to get on the same page with imports so they'll stop getting shuffled around everytime someone runs an import optimiser when they alter code.. it just makes the diffs more complex than they need to be.

That being said, fallowing the Team-RTG style rules, java imports should remain at the top.
```
javax.*
java.*
<blank line>
all other imports
<blank line>
static imports
```